external-applications/quarkus-workshop-super-heroes/src/test/resources/hero.yaml

```diff
@@ -23,12 +23,18 @@ items:
       git:
         uri: https://github.com/quarkusio/quarkus-workshops.git
       type: Git
+      configMaps:
+      - configMap:
+          name: settings-mvn
+        destinationDir: "configuration"
     strategy:
       type: Source
       sourceStrategy:
         env:
         - name: ARTIFACT_COPY_ARGS
           value: -p -r lib/ *-runner.jar
+        - name: MAVEN_ARGS
+          value: -s /opt/app-root/src/configuration/settings.xml -Dquarkus-plugin.version=${version.plugin.quarkus} -Dquarkus.platform.version=${version.quarkus} -Dquarkus.platform.group-id=${quarkus.platform.group-id} -Dquarkus.platform.artifact-id=${quarkus.platform.artifact-id} -DskipTests=true
```

Comment:
Adding `-s /opt/app-root/src/configuration/settings.xml` is a workaround as the instructions I've found for overwriting the `settings.xml` in OCP didn't work. 

Also I have to skip the tests as they are using docker.